estatioapp/app/src/main/java/org/estatio/module/assetfinancial/contributions/FinancialAccount_newAccount.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ *  Copyright 2012-2014 Eurocommercial Properties NV
+ *
+ *
+ *  Licensed under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.estatio.module.assetfinancial.contributions;
+
+import javax.inject.Inject;
+
+import org.apache.isis.applib.annotation.Action;
+import org.apache.isis.applib.annotation.DomainService;
+import org.apache.isis.applib.annotation.MemberOrder;
+import org.apache.isis.applib.annotation.NatureOfService;
+import org.apache.isis.applib.annotation.SemanticsOf;
+
+import org.estatio.module.asset.dom.FixedAsset;
+import org.estatio.module.assetfinancial.dom.FixedAssetFinancialAccount;
+import org.estatio.module.financial.dom.FinancialAccount;
+
+@DomainService(
```

Comment:
This refactoring is incomplete - needs @Mixin(method="act") here. instead of @DomainService